<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bsrForm</class>
 <widget class="QDialog" name="bsrForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BSR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Send BSR (latest on top)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="bsrComboBox"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="infoTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="easypalPushButton">
       <property name="text">
        <string>Easypal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compatiblePushButton">
       <property name="text">
        <string>Compatible</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
